packages/web/src/config/theme.ts

```diff
@@ -0,0 +1,13 @@
+export const theme = {
+  colors: {
+    main: "#98D2EB", // light blue
+    background1: "#FFFFFF", // white
+    background2: "#274690", // dark blue
+    positive: "#274690", // green
+    negative: "#F15025", // red
+    border: "#EAEAFF", // gray
+    borderActive: "#274690", // gray
+    contrast1: "#000505", // black
+    contrast2: "#FFFFFF" // black
+  }
+};
```

Comment:
Keep the theme's variables in a file to make it easy to change colours globally.